gitlab-pipeline/stage/publish.yml

```diff
@@ -141,3 +141,16 @@ publish:acceptance:raspberrypi4:
     - test:acceptance:raspberrypi4
   variables:
     JOB_BASE_NAME: raspberrypi4
+
+publish:mender-dist-packages:
+  stage: publish
+  rules:
+    - if: '$INTEGRATION_REV =~ /^[0-9]+\.[0-9]+\.[0-9]+(-build\d+)?$/'
+  variables:
```

Comment:
I don't know what you think @kacf , but I am considering skipping the tests in mender-dist-packages when triggering from here

The reasons are 1) that any recipe and source code change is being tested in CI on every commit and 2) that we might actually need to skip the tests for bugfix releases (see [this variable](https://github.com/mendersoftware/mender-dist-packages/blob/master/.gitlab-ci.yml#L15)).

On the other hand, by doing so we are accepting some risk in the process, so I am not very confident neither.

(right, this was mostly thinking out loud)

I suggest we leave it as is, but keep in mind that we might need to add a conditional skip here for bugfix releases down the road if we ever introduce again breaking changes in the tests.

Reply:
Indeed, this is quite tricky. I think the best way forward would be to fix [QA-123](https://tracker.mender.io/browse/QA-123). Maybe we should bump the priority of this one a little bit to make a more robust release pipeline?

In any case, I agree leaving this as is for now. Better to have a false red build, than a false green build.